travis/scripts/01-generate-project.sh

```diff
@@ -5,13 +5,7 @@ set -ev
 #-------------------------------------------------------------------------------
 mv -f $JHIPSTER_SAMPLES/$JHIPSTER $HOME/
 cd $HOME/$JHIPSTER
-if [ $GRUNT == 1 ]; then
-  rm -Rf $HOME/$JHIPSTER/node_modules/.bin/*gulp*
-  rm -Rf $HOME/$JHIPSTER/node_modules/*gulp*
-else
-  rm -Rf $HOME/$JHIPSTER/node_modules/.bin/*grunt*
-  rm -Rf $HOME/$JHIPSTER/node_modules/*grunt*
```

Comment:
Yes exact! There was an issue in the past, about using cache for node_modules, I don't remember exactly. So to secure this PR, until I have time to clean https://github.com/jhipster/jhipster-travis-build with grunt lib